### PR TITLE
Converter to qpolygon

### PR DESCRIPTION
**Pull request by mentonin**
## Summary of Changes

Adds converter from `std::list<K::Point_2>` to `QPolygonF` that was documented but not implemented. Also fixes documentation for the converter in the opposite direction to match the actual implementation and adds documentation for the converter from `K::Circular_arc_point_2` to `QPointF`.

## Release Management

* Affected package(s): Qt

### CODE REVIEW

File: GraphicsView/doc/GraphicsView/CGAL/Qt/Converter.h [hunk 1]
```diff
@@ -41,6 +41,11 @@ Converts a point.
 */
 QPointF operator()(K::Point_2);
 
+/*!
+Converts a circular arc point.
+*/
+QPointF operator()(K::Circular_arc_point_2)
+
```
Comment (lrineau) [inline GraphicsView/doc/GraphicsView/CGAL/Qt/Converter.h:48]:
```suggestion
/*!
Converts a circular arc point.
*/
QPointF operator()(K::Circular_arc_point_2);

```

**Review by lrineau — CHANGES_REQUESTED** [review]
Thanks for your contribution.

I found out two errors (typos) that are easy to fix, before this pull-request can be accepted.

**Comment by mentonin** [comment]
Fixed typos

**Comment by sloriot** [comment]
Successfully tested in CGAL-5.6-Ic-223

**Pull request merged**